conda/scripts/get_freecad_version.py

```diff
@@ -25,10 +25,10 @@
 date = str(datetime.now()).split(" ")[0]
 
 version_info = FreeCAD.Version()
-dev_version = version_info[0] + "." + version_info[1] + "." + version_info[2]
+build_version_suffix = FreeCAD.ConfigDump()["BuildVersionSuffix"]
```

Comment:
nitpick but this seems more straightforward ;)
```suggestion
build_version_suffix = FreeCAD.ConfigGet("BuildVersionSuffix")
```